pages/validators/requirements.md

```diff
@@ -17,4 +17,34 @@ For your node to successfully ingest bridge transactions from the relevant block
 💡IMPORTANT💡:The RPC endpoint you choose *MUST* satisfy the following requirements
 * supports `eth_chainId` method
 * supports `eth_getLogs` method
-* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
\ No newline at end of file
+* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
+
+# Required Configs
+
+💡IMPORTANT💡 The dYdX Chain has important node configurations required for normal chain operation. This includes:
```

Comment:
This is already in a page called `Required`; I don't think we need to call out individual line items as important